examples/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <title>Adobe Data Layer | Examples</title>
     <script>dataLayer = [];</script>
-    <script src="../dist/datalayer.js" async></script>
+    <script src="../src/scripts/datalayer.js" async></script>
```

Comment:
@jckautzmann - why use the source here?

Reply:
I used it while testing, to have the change right away in the browser, without having to first build

Reply:
Ok, it should be built anyway with the default watch task in `gulp`, and when we merge https://github.com/adobe/adobe-client-data-layer/pull/5 I don't think this will work.